### ISSUE TITLE
Write new e2e test: Shopper / Subscriptions / Purchase free trial subscription

### ISSUE THREAD

**Issue by veljkho** [Automattic/woocommerce-payments#2221]
Write a new e2e test that covers the following flow:
- Shopper / Subscriptions / Purchaser free trial subscription

**Comment by dechov** [comment]
Having run into a couple high priority issues specifically with free trials, this one would be great to have in. $0 orders use a bit of a separate flow in the code, especially when SCA is required since it creates/uses a "Setup Intent" rather than a "Payment Intent".

On that note: if only one card is used in this test, IMO it should probably be `4000002500003155` (the top one in [this doc section](https://stripe.com/docs/testing#regulatory-cards)), so that we can verify that it's set up properly with SCA and that the renewal works.

**Comment by rodelgc** [comment]
Hi @veljkho , @dechov 

> so that we can verify that it's set up properly with SCA and that the renewal works.

I noticed that when the customer is in free trial, they don't see the "Renew now" button. So I'm not sure how we could test the renewal here.

![Screen Shot 2021-07-31 at 2.02.56 AM.png](https://images.zenhubusercontent.com/60a2535d3762da5e2703d6dd/3461aa91-c547-4f7d-a177-e0a05b2a1f33)

Anyway, I'm now submitting the PR for this flow. I used the 3DS card that @dechov mentioned above. Thanks!
